doc/OWNER.md

```diff
@@ -0,0 +1,8 @@
+# WasmEdge Maintainers
+
+| Maintainer | GitHub ID | Company | Email |
+| --------------- | --------- | ----------- | ----------- |
+| Michael Yuan | @juntao | Second State | <michael@secondstate.io> |
+| Hydai Tai | @Hydai | Second State | <hydai0@secondstate.io> |
+| Yi-Ying He | @q82419 | Second State| <yiying@secondstate.io> |
```

Comment:
You miss a space after the `Second State`

Reply:
fixed